README.md

```diff
@@ -1,2 +1,30 @@
 # ColorClassifer-TensorFlow.js
 Community version of TensorFlow.js Color Classifier
+
```

Comment:
Could you do this with markdown instead? i.e.

```
* [link title](link url)
```

Reply:
I'm on it